app/Http/Controllers/HiresController.php

```diff
@@ -81,14 +86,17 @@ public function store(){
         foreach ($steps as $step) {
             HireStep::create([
                 "hire_id" => $hire->id,
-                "step_id" => $step->id
+                "step_id" => $step->id,
+                "step_name" => $step->name
             ]);
         }
 
         // Add into the hire_locks table
         HireLock::create([
             "hire_id" => $hire->id
         ]);
+
+        User::first()->notify(new NewHireAdded($hire));
```

Comment:
As of now the user may or may not have a manager at this point. All of the notifications are flowing into the same Slack channel. If manager is required at submission it could be changed so that each user has a preferred slack channel and the messages would go there. We would just have to add the field in the database for webhook url

Reply:
Switched over to this format to account for not having a manager assigned
```
if($hire->manager_id == null){
    User::first()->notify(new NewHireAdded($hire));
} else {
    User::find($hire->manager_id)->notify(newHireAdded($hire));
}
```